### ISSUE TITLE
 False-positive in URLCONNECTION_SSRF_FD

### ISSUE THREAD

**Issue by Vampire** [find-sec-bugs/find-sec-bugs#505]
With this code:
```java
public class Version {
    private static final URL versionPropertiesResource =
            Version.class.getResource("../version.properties");
    public Version() {
        Properties versionProperties = new Properties();
        try (InputStream versionPropertiesStream = versionPropertiesResource.openStream()) {
            versionProperties.load(versionPropertiesStream);
        } catch (IOException ignored) {
        }
    }
}
```
`URLCONNECTION_SSRF_FD` complains about using `openStream()` on a user-supplied URL which clearly is not the case. (It is in a private static final field to be able to easily replace it for testing)

**Comment by topolik** [comment]
Should be fixed by #490

**Comment by Vampire** [comment]
Too sad that there are no snapshot builds published.
But I built locally and tried it out, works indeed with latest `master` code.
I hope it will be released somewhen :-)
